view/desktop/bin/src/commands/cmd_dummy.rs

```diff
@@ -19,7 +20,14 @@ pub async fn fetch_all_themes() -> Result<Vec<String>, String> {
 #[tauri::command(async)]
 #[specta::specta]
 pub async fn read_theme(theme_name: String) -> Result<String, String> {
-    let theme_file_path = PathBuf::from(std::env::var("HOME").unwrap())
+    let home_dir = if cfg!(target_family = "unix") {
+        PathBuf::from(std::env::var("HOME").unwrap())
+    } else {
+        my_home()
+            .unwrap()
+            .expect("Failed to retrieve the home directory")
+    };
```

Comment:
I think, here is better to use 

```rust
fn get_home_dir() -> Result<PathBuf, String> {
    #[cfg(target_os = "windows")]
    {
        windows_home_dir()
    }

    #[cfg(any(target_os = "macos", target_os = "linux"))]
    {
        unix_home_dir()
    }

    #[cfg(not(any(target_os = "windows", target_os = "macos", target_os = "linux")))]
    {
        Err("Unsupported operating system".to_string())
    }
}

/// Retrieves the home directory on Unix-like systems (macOS and Linux).
#[cfg(any(target_os = "macos", target_os = "linux"))]
fn unix_home_dir() -> Result<PathBuf, String> {
    std::env::var("HOME")
        .map(PathBuf::from)
        .map_err(|e| format!("Failed to retrieve HOME environment variable: {}", e))
}


/// Retrieves the home directory on Windows.
#[cfg(target_os = "windows")]
fn windows_home_dir() -> Result<PathBuf, String> {
    ...
}
```

Use example

```rust
 match get_home_dir() {
        Ok(home_dir) => todo!(),
        Err(err) =>  todo!(),,
    }
```

Reply:
Great idea! I was thinking about making it into a function, but I'm not sure where I should put it. What's your thought?